peru/resources/plugins/bzr/bzr_plugin.py

```diff
@@ -0,0 +1,85 @@
+#! /usr/bin/env python3
+
+import os
+import os.path
+import shutil
+import subprocess
+
+
+def bzr(path, *args, capture_output=False):
+    cwd = os.getcwd()
+    os.chdir(path)
+
+    try:
+        command = ['bzr']
+        command.extend(args)
+        stdout = subprocess.PIPE if capture_output else None
+        process = subprocess.Popen(command, stdin=subprocess.DEVNULL,
+                                   stdout=stdout, universal_newlines=True)
+        output, _ = process.communicate()
+        if process.returncode != 0:
+            raise RuntimeError(
+                'Command exited with error code {0}:\n$ {1}\n{2}'.format(
+                    process.returncode,
+                    ' '.join(command),
+                    output))
+        return output
+    finally:
+        os.chdir(cwd)
+
+
+def clone_if_needed(bzr_path, url):
+    if os.path.exists(os.path.join(bzr_path, '.bzr')):
+        return False
+    try:
+        bzr(bzr_path, 'branch', '--use-existing-dir', url, '.',
```

Comment:
With git and hg, we clone bare repositories here, because otherwise the working tree will just take up space. Would bzr's `--no-tree` flag do that for us here?